File: Code Forces/PY/Football.py
n = int(input())

items = []
for i in range(n):
	team = input()
	items.append(team)

teams = set(items)
maxt = 0
winner = ""
for team in teams:
	if items.count(team) >= maxt:
		maxt = items.count(team)
		winner = team


print(winner)

File: Code Forces/CPP/Football.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
	int n;
	string team;
	map<string, int>teams;
	cin>>n;
	for (int i = 0; i < n; ++i){
		cin>>team;
		teams[team]+=1;
	}

	cout<<max(teams)<<endl;
	return 0;
}